runner/src/server/plugins/engine/views/components/clientsidefileuploadfield.html

```diff
@@ -261,9 +272,48 @@
         errorSpan.textContent = errorMessage;
     };
 
+    // Show error summery with the error message in js side
+    const handleErrorSummery = (errorMessage) => {
+        const errorSummeryParentContainer = document.querySelector('.govuk-grid-column-two-thirds');
+        const errorSummery = document.querySelector('[data-module="govuk-error-summary"]');
+        if(!errorSummery && errorSummeryParentContainer) {
+            errorSummeryParentContainer.insertAdjacentHTML('afterBegin', `
+                <div class="govuk-error-summary" aria-labelledby="error-summary-title" role="alert" tabindex="-1" data-module="govuk-error-summary">
+                    <h2 class="govuk-error-summary__title" id="error-summary-title">
+                        There is a problem
+                    </h2>
+                    <div class="govuk-error-summary__body">
+                        <ul class="govuk-list govuk-error-summary__list">
+                            <li>
+                                <a href="#`+"{{component.model.id}}"+`">`+errorMessage+`</a>
+                            </li>
+                        </ul>
+                    </div>
+                </div>
+        `);
+        } else if (errorSummery) {
+            const errorMessageLoc = errorSummery.querySelector(".govuk-error-summary__body > ul > li > a");
+            if (errorMessageLoc) {
+                errorMessageLoc.innerHTML = errorMessage;
+            }
+        }
+    };
```

Comment:
Maybe it's better to have this error in the html and hidden via css, then we change the css/contents with JavaScript alone?  Rather than a bigger chunk of JavaScript with the html embedded inside a string.  Just a thought

Reply:
yes @tferns I agree with you but the issue is we are doing a some work around here if we add a above html into the components it will not act as we wanted since it goes under the components title only thing we can do it adding it to the parent component as a hidden I just  don't wanted to touch it since it is the parent for every thing so we'll have a call and discuss this